cdk/apps/slack/stacks/slack_lambda.py

```diff
@@ -77,17 +77,34 @@ def __init__(self, scope: core.Construct, id: str, slack_channel: str, **kwargs)
             role=lambda_role
         )
 
-        rule = _events.Rule(
-            self,
-            'BatchEventToSlackLambda'
-        )
-        rule.add_event_pattern(
-            source=['aws.batch'],
-            detail_type=['Batch Job State Change'],
-            detail={'status': [
-                'FAILED',
-                'SUCCEEDED',
-                'RUNNABLE'
-            ]}
-        )
-        rule.add_target(_events_targets.LambdaFunction(handler=function))
+        batch_job_queues = {
+            'AGHA':           {'name': 'agha-job-queue',                         'enabled': True},
+            'Umccrise':       {'name': 'cdk-umccrise_job_queue',                 'enabled': True},
+            'UmccriseDragen': {'name': 'cdk-umccrise_job_queue-dragen-testing',  'enabled': False},
+            'Nextflow':       {'name': 'nextflow-job-queue',                     'enabled': False},
+            'WtsReport':      {'name': 'wts_report_batch_queue_dev',             'enabled': True},
+        }
+        for job_queue_id, job_queue_config in batch_job_queues.items():
+            aws_account_id = kwargs['env']['account']
+            aws_region = kwargs['env']['region']
```

Comment:
Looking at the CDK docs I wasn't able to see an obvious way to reference the `region` and `account` variables that are passed to the `Stack` constructor:
https://github.com/umccr/infrastructure/blob/cb80fd0f7096b8e76e932d592d4b48d91e020832/cdk/apps/slack/app.py#L20-L25

Using `kwargs` doesn't seem like the most robust approach. Any suggestions?